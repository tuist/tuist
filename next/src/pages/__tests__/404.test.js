import React from 'react'
import renderer from 'react-test-renderer'
import NotFoundPage from '../404'

describe('NotFoundPage', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NotFoundPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
