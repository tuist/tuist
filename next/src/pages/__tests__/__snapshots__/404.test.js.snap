// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<main
  style={
    Object {
      "color": "#232129",
      "fontFamily": "-apple-system, Roboto, sans-serif, serif",
      "padding": "96px",
    }
  }
>
  <title>
    Not found
  </title>
  <h1
    style={
      Object {
        "marginBottom": 64,
        "marginTop": 0,
        "maxWidth": 320,
      }
    }
  >
    Page not found
  </h1>
  <p
    style={
      Object {
        "marginBottom": 48,
      }
    }
  >
    Sorry
     
    <span
      aria-label="Pensive emoji"
      role="img"
    >
      ðŸ˜”
    </span>
     
    we couldnâ€™t find what you were looking for.
    <br />
    <br />
    <a
      href="/"
    >
      Go home
    </a>
    .
  </p>
</main>
`;
